app/uk/gov/hmrc/vatapi/resources/VatReturnsResource.scala

```diff
@@ -16,25 +16,17 @@
 
 package uk.gov.hmrc.vatapi.resources
 
+import cats.implicits._
```

Comment:
Why are we using `cats` library imports here?

Reply:
The controller will not work if removed